library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -1313,6 +1313,9 @@ impl<T, A: Allocator> VecDeque<T, A> {
     /// If [`make_contiguous`] was previously called, all elements of the
     /// deque will be in the first slice and the second slice will be empty.
     ///
+    /// Note that the particular split point between the slices is not guaranteed
+    /// and may change based on internal implementation details.
+    ///
```

Comment:
I think the addition could be a part of the previous paragraph since it's talking about the same thing. E.g.

> If [`make_contiguous`] was previously called ... and the second slice will be empty. Otherwise, the exact split point depends on implementation details and is not guaranteed.